context/app/static/js/components/entity-search/sidebar/ConfigureSearchTable/ConfigureSearchTable.jsx

```diff
@@ -27,29 +27,31 @@ function ConfigureSearchTable({
         <TableHead>
           <TableRow>
             <TableCell />
-            <TableCell>Add Facet</TableCell>
-            <TableCell>Add Column</TableCell>
+            <NoWrapIconTooltipCell tooltipTitle="Selecting this checkbox will add the term as a facet for the search results.">
+              Add Facet
+            </NoWrapIconTooltipCell>
+            <NoWrapIconTooltipCell tooltipTitle="Selecting this checkbox will add the term as a column for the search results table.">
+              Add Column
+            </NoWrapIconTooltipCell>
           </TableRow>
         </TableHead>
         <TableBody>
           {filteredFields.map(([fieldName, fieldConfig]) => (
             <TableRow key={fieldName}>
-              <IconTooltipCell tooltipTitle={metadataFieldDescriptions[fieldConfig?.ingestValidationToolsName]}>
+              <StyledIconTooltipCell tooltipTitle={metadataFieldDescriptions[fieldConfig?.ingestValidationToolsName]}>
                 {fieldConfig.label}
-              </IconTooltipCell>
-              {['string', 'boolean'].includes(fieldConfig.type) || numericFacetsProps?.[fieldName] ? (
-                <TableCell>
+              </StyledIconTooltipCell>
+              <TableCell padding="checkbox" align="center">
```

Comment:
The `checkbox` confused me, but it is a special option: https://mui.com/material-ui/api/table-cell/ ... Cool.